SwissKnife/gcloud/GCloudStorage.py

```diff
@@ -267,7 +268,7 @@ def list_blobs(self,
                                                           with_gs=False)
         return self.bucket.list_blobs(prefix=list_prefix)
 
-    def copy_blob(self, src_blob: Blob, dst_storage: "GCloudStorage", dst_file_name: str) -> Blob:
```

Comment:
To be more specific with `dst_storage` parameter, you can import `from google.cloud.storage.bucket import Bucket` and then specify type in `dst_storage` like this: `dst_storage: Bucket`

Reply:
That param is not a bucket from Google Library. It is the GCloudStorage from SwissKnife.
I can't use it as type hinting from itself (this is a method of GCloudStorage object). For that reason, I used a explicit string.

Reply:
True! I was confused with the gcloud copy function... sorry for the misunderstanding!